pkg/controllers/management/restrictedadminrbac/cluster.go

```diff
@@ -0,0 +1,122 @@
+package restrictedadminrbac
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/go-multierror"
+	apisv3 "github.com/rancher/rancher/pkg/apis/management.cattle.io/v3"
+	v3 "github.com/rancher/rancher/pkg/apis/management.cattle.io/v3"
```

Comment:
I might be missing something, but this looks like the same import aliased to different things. You mind explaining why this is here?

Reply:
Fixed.